vueui/src/assets/icons.scss

```diff
@@ -267,4 +267,4 @@ $fa-css-prefix: "ic";
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
You did not change this file otherwise, so this change really shouldn't be in the commit.